### PR TITLE
[Core][Fixture] Product reviews fixtures

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | yes |
| BC breaks?      | no |
| Related tickets | |
| License         | MIT |

It would be nice to have some product reviews created as sample data ;)

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml [hunk 1]
```diff
@@ -156,6 +156,10 @@ sylius_fixtures:
                     options:
                         amount: 15
 
+                reviews:
```
Comment (TheMadeleine) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml]: Aren't these `product_reviews`? :)

Comment (pjedrzejewski) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml]: 👍

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml [hunk 2]
```diff
@@ -156,6 +156,10 @@ sylius_fixtures:
                     options:
                         amount: 15
 
+                product_reviews:
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml]: `product_review` to be consistent with our convention in this file

**Comment by pjedrzejewski** [comment]
Thank you Mateusz!

**Pull request merged**